swagger.yaml

```diff
@@ -5175,6 +5175,8 @@ definitions:
           type: string
       speed_mbps:
         type: integer
+      type:
+        type: string
```

Comment:
Maybe it should be enum ?
Where is it used?

Reply:
This will be used to determine if we should display or filter the interface.

Right now we only really care about if it is "physical" or not.
This was also discussed in https://github.com/openshift/assisted-installer-agent/pull/360#issuecomment-1138910540